apps/hosts/urls.py

```diff
@@ -24,9 +24,9 @@
         name="get_challenge_host_list",
     ),
     url(
-        r"^challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/challenge_host/(?P<pk>[0-9]+)$",
-        views.challenge_host_detail,
-        name="get_challenge_host_details",
+        r"^challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/challenge_host/(?P<challenge_host_pk>[0-9]+)$",
+        views.challenge_host_delete,
+        name="get_challenge_host_delete",
```

Comment:
Thanks for your thoughtfulness in changing the names. I agree the old names are somewhat unclear.
But I think the main purpose of the PR (to add feature to delete host/participant) is already almost complete. I think a different PR for renaming would be more convenient and for this PR, we can just focus on adding the feature.
Again, thanks for the thoughtfulness. :)

Reply:
@nikochiko  All the suggested changes has been done and will open a new PR regarding renaming as soon as possible.Thanks a lot for your suggestions.I helped me a lot to improve my code structure and functionality.

Reply:
Perfect!